cbor/src/main/java/com/fasterxml/jackson/dataformat/cbor/CBORGenerator.java

```diff
@@ -236,6 +253,11 @@ public int getMask() {
      */
     protected boolean _bufferRecyclable;
 
+    /**
+     * Table of previously referenced text and binary strings.
+     */
```

Comment:
... Used if StringRef functionality is enabled. // or something like that

`@since 2.15`

Reply:
I wasn't sure if protected fields should be marked, I'll go through and try to add it for the fields I've added. (as well as the comment clarification noted here)